tests/ui/non_expressive_names.rs

```diff
@@ -134,3 +134,10 @@ fn bla() {
         }
     }
 }
+
+fn underscores_and_numbers() {
+    let _1 = 1; //~ERROR Consider a more descriptive name
+    let ____1 = 1; //~ERROR Consider a more descriptive name
+    let __1___2 = 12; //~ERROR Consider a more descriptive name
+    let _1_ok= 1;
+}
```

Comment:
missing newline at end of file

Reply:
I am super confused about this one, I did change it and it has an empty line on my machine, git diff shows nothing. I will figure it out tomorrow